src/components/page-environment/replicas.js

```diff
@@ -1,14 +1,31 @@
 import { smallReplicaName } from '../../utils/string';
+import * as routing from '../../utils/routing';
+import { Link } from 'react-router-dom';
+import { Typography } from '@equinor/eds-core-react';
 
-const Replicas = ({ replicaList }) => (
+const Replicas = ({ appName, envName, componentName, replicaList }) => (
   <>
-    {replicaList &&
+    {replicaList ? (
       replicaList
-        .map((replica) => (
-          <span key={replica.name}>{smallReplicaName(replica.name)}</span>
+        .map((replica, i) => (
+          <Link
+            key={i}
+            to={routing.getReplicaUrl(
+              appName,
+              envName,
+              componentName,
+              replica.name
+            )}
+          >
+            <Typography link as="span">
+              {smallReplicaName(replica.name)}
+            </Typography>
+          </Link>
         ))
-        .reduce((prev, curr) => [prev, ', ', curr])}
-    {!replicaList && <span key="none">No active replicas</span>}
+        .reduce((prev, curr) => [prev, ', ', curr])
```

Comment:
The reduce works, but it builds an array where the first element is an array which first element is also an array... (img 1) and so on. This is probably ok, and React renders it correctly, but if you want to create a flat array you need to check if the value of prev is an array (img 2). `[...(Array.isArray(prev) ? prev : [prev]), ', ', curr]`
![image](https://user-images.githubusercontent.com/65334626/130433294-7cdde64b-aea7-4044-a1e9-2c5aa7b22635.png)

![image](https://user-images.githubusercontent.com/65334626/130433681-443facd5-d716-4322-b79d-bc297cada810.png)